viewing_party/party.py

```diff
@@ -1,23 +1,176 @@
 # ------------- WAVE 1 --------------------
 
+# from tkinter.tix import InputOnly
+
+
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+        # validate input
+    new_dict = {}
+    new_dict["title"] = title
+    new_dict["genre"] = genre
+    new_dict["rating"] = rating
+    # if not new_dict["key"]
+    # for key in new_dict:
+    #     if not new_dict[key]:
+    #         return None
+    return new_dict
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            i = user_data["watchlist"].index(movie)

```

Comment:
This works just fine, but we can use remove with the item since we know the item exists (since we're iterating over them directly). If we're worried about modifying the list we're iterating over, it's fine to add a `break` like so:
```python
...
user_data["watchlist"].remove(movie)
break
```